experiments/profile_data.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+
+
+class ProfileData:
```

Comment:
This comment applies to the whole class:
My preference is that we do not use the generic terms config/measurement. For clarity, I think it pays to be a little more verbose and call these what they are run_config/run_config_measurement. 
This would apply to method names, arguments, and internal variables.

Reply:
This doesn't have anything to do with file names - it has to do with accurately representing the data/class that is being used/passed in. Due to multi-model, we have different types of measurements: run_config and model_config, and I just think we need to be explicit with our naming (even though I know that measurement will refer to run_config_measurement 99% of the time).

Reply:
I've cleaned up the variable names and have also renamed the classes